doc/build-openbsd.md

```diff
@@ -12,11 +12,10 @@ Preparation
 Run the following as root to install the base dependencies for building:
 
 ```bash
-pkg_add git gmake libevent libtool
-pkg_add autoconf # (select highest version, e.g. 2.69)
-pkg_add automake # (select highest version, e.g. 1.15)
-pkg_add python # (select highest version, e.g. 3.6)
-pkg_add boost
+pkg_add git gmake libevent libtool boost
```

Comment:
I don't think we need to specify minor/patch versions here. They are just more numbers that will quickly go "out of date"

Reply:
Updated. Please re-review :-)